app/controllers/trn_requests_controller.rb

```diff
@@ -30,9 +30,15 @@ def update
            Faraday::TimeoutError,
            DqtApi::TooManyResults,
            TrnHasAlert
-      create_zendesk_ticket
-
-      redirect_to helpdesk_request_submitted_path
+      begin
+        create_zendesk_ticket
+        redirect_to helpdesk_request_submitted_path
+      rescue ZendeskService::ConnectionError => exception
+        CreateZendeskTicketJob.set(wait: 1.day).perform_later(trn_request.id)
```

Comment:
1 day seems like a long wait before retrying again, in the case of something like a small flake in the API. Could it be maybe 5 minutes, with exponential backoff? (You mentioned it should do this by default in the PR description)